src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1350,6 +1360,8 @@ private TextLoader(IHost host, ModelLoadContext ctx)
 
             // REVIEW: Should we serialize this? It really isn't part of the data model.
             _useThreads = true;
+            // MYTODO: Should we serialize this? probably yes and also include it in Flags
+            _readMultilines = false;
```

Comment:
If we decide to keep _readMultilines as a flag, then I'll also need to serialize it.